cmd/gateway/panfs/gateway-panfs.go

```diff
@@ -73,12 +76,18 @@ func panfsGatewayMain(ctx *cli.Context) {
 		cli.ShowCommandHelpAndExit(ctx, minio.PANFSBackendGateway, 1)
 	}
 
-	minio.StartGateway(ctx, &PANFS{ctx.Args().First()})
+	var metaBucketPath string = env.Get(
+		"MINIO_PANFS_METABUCKET_PATH",
+		"/var/lib/minio",
+	)
+
+	minio.StartGateway(ctx, &PANFS{ctx.Args().First(), metaBucketPath})
 }
 
 // PANFS implements Gateway.
 type PANFS struct {
-	path string
+	path           string
```

Comment:
How do we use original fsPath in PanFS backend after all changes?

Reply:
I mean maybe the easiest thing is to do the same thing, but without MinIO changes and it running script (load_minio), that can provide path to /var/lib/minio instead of path on PanFS?